src/Components/Main/App/Content/Users/Users.js

```diff
@@ -17,9 +17,9 @@ export default class Users extends React.Component {
 	}
 
 	getAllUserData() {
-		let response = this.props.getRequest("http://localhost:2000/users/");
-		console.log(response);
-		return response;
+		this.props.getRequest("http://localhost:2000/users/", (res) => {
```

Comment:
Just need to change it there and send token with url. Query string or something?